src/style.css

```diff
@@ -0,0 +1,78 @@
+.html {
+  font-family: Avenir, sans-serif;
+}
+
+.input {
+  width: 100%;
+  margin-bottom: 20px;
+  font-weight: 300;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  color: #3d4e61;
+  font-family: inherit;
+  background-image: url("./images/Search.svg");
+  background-repeat: no-repeat;
+  box-sizing: border-box;
+}
+
+.form:first-child {
+  margin-top: 20px;
+}
+
+.form:last-child {
+  margin-bottom: 0;
+}
+
+.big_search {
+  
+  background-position: left 25px top 24px;
+  
+  padding-left: 62px;
+  background-size: 19px 19px;
+  height: 70px;
+  line-height: 22px;
+  font-size: 16px;
+}
+
+.small_search {
```

Comment:
don't add extra classes

make it like that
```
.search {
  all styles
}

.small_search {
  styles that differ from `.search`
}

Reply:
no need to use small_search and big_search

Reply:
<img width="342" alt="image" src="https://user-images.githubusercontent.com/102411130/223364913-39b66b11-64a5-4192-98a5-c0108d096f65.png">